CONTRIBUTING.md

```diff
@@ -1,94 +1,125 @@
 Contributing
 ============
 
-In this document: 
+Thank you for your interest in contributing to the .NET documentation!
```

Comment:
Here, I see an article before ".NET documentation". Later in the doc, I see it w/o. It feels more natural to me w/o, but we should probably be consistent.